src/utils/paginateFunction.tsx

```diff
@@ -0,0 +1,6 @@
+export function paginate(ResponseArray: any, page_size: number, page_number: number) {
```

Comment:
paginate function should receive `records` of type `any` and not `ResponseArray` as parameter

Reply:
Convert paginate to be generic, so that we don't loose typing.

```js
export function paginate<T>(records: T[], page_size: number, page_number: number): T[] {
    return records.slice(
      (page_number - 1) * page_size,
      page_number * page_size
    );
}
```